### ISSUE TITLE
Twenty Fourteen: full width pages not taking up full width

### ISSUE THREAD

**Issue by Nic-Sevic** [Automattic/themes#3645]
## Steps to replicate

Add the Twenty Fourteen theme to any site
See that extra space appears on right side and remaining content no longer full width

## Result
ends up with extra space on right

![does](https://user-images.githubusercontent.com/47717887/114517563-20183300-9c47-11eb-99d6-3efcfad6886c.jpeg)

## Expected

Should look like this: 

![should](https://user-images.githubusercontent.com/47717887/114517469-0aa30900-9c47-11eb-8fdf-bde7a2dc7be7.jpeg)

* User report: 3899850-zen, 28899642-hc
* Follow-up ticket: [shortlink to ticket, ie 12345-zen]

This CSS should fix temporarily:
```
* fix full width bug in 20 14 | xxxxxxx-zen NS*/
#page, #site-header img, #masthead {
	min-width: 100vw;
}
```

**Comment by kathrynwp** [comment]
The narrow appearance of this core bundled theme is by design, and not a bug.

**Comment by mriyazuddin** [comment]
I have a user reporting this here ( #6315539 ). I was able to replicate this issue on a test site. The CSS workaround provided above works. I have shared that with user. This is what I have tried:

* Disabled all plugins.
* Flushed cache.
* Deleted and reinstalled Gutenberg to force the new version.
* No existing CSS code on the site.
* Created a new page copying the content.
* Changed the homepage, still no go. Changed it back to the original homepage.
* Was also able to replicate this on a test site.

https://github.com/Automattic/themes/assets/49064126/c0494aa1-3be2-40ed-8f28-8c409b0c997f
